tools/wasp-cli/chain/accounts.go

```diff
@@ -157,6 +157,8 @@ func initDepositCmd() *cobra.Command {
 						},
 					)
 				})
+
+				fmt.Print(res.Digest.String())
```

Comment:
If this print is for the user, then maybe add something like `"Digest: "`? Also use `log.Printf` for consistency? And/or `log.Verbosef`, `log.DebugFlag`, etc.